cpp/src/io/json/nested_json.hpp

```diff
@@ -199,7 +201,27 @@ struct json_column {
                   uint32_t child_count)
```

Comment:
cab we move these function definition to `nested_json_gpu.cu` or `.cpp`?

Reply:
Good point. Moved definition to `nested_json_gpu.cu`